src/components/Calculator.js

```diff
@@ -1,30 +1,46 @@
-        <button type="button">AC</button>
-        <button type="button">+/-</button>
-        <button type="button">%</button>
-        <button type="button" className="orange">/</button>
-        <button type="button">7</button>
-        <button type="button">8</button>
-        <button type="button">9</button>
-        <button type="button" className="orange">x</button>
-        <button type="button">4</button>
-        <button type="button">5</button>
-        <button type="button">6</button>
-        <button type="button" className="orange">-</button>
-        <button type="button">1</button>
-        <button type="button">2</button>
-        <button type="button">3</button>
-        <button type="button" className="orange">+</button>
-        <button type="button" className="zero">0</button>
-        <button type="button">.</button>
-        <button type="button" className="orange">=</button>
+        <div className="screen">{next || total}</div>
+        <button type="button" onClick={(e) => this.eventHandler(e.target.textContent)}>AC</button>
+        <button type="button" onClick={(e) => this.eventHandler(e.target.textContent)}>+/-</button>
+        <button type="button" onClick={(e) => this.eventHandler(e.target.textContent)}>%</button>
+        <button type="button" onClick={(e) => this.eventHandler(e.target.textContent)} className="orange">/</button>
+        <button type="button" onClick={(e) => this.eventHandler(e.target.textContent)}>7</button>
+        <button type="button" onClick={(e) => this.eventHandler(e.target.textContent)}>8</button>
+        <button type="button" onClick={(e) => this.eventHandler(e.target.textContent)}>9</button>
+        <button type="button" onClick={(e) => this.eventHandler(e.target.textContent)} className="orange">x</button>
+        <button type="button" onClick={(e) => this.eventHandler(e.target.textContent)}>4</button>
+        <button type="button" onClick={(e) => this.eventHandler(e.target.textContent)}>5</button>
+        <button type="button" onClick={(e) => this.eventHandler(e.target.textContent)}>6</button>
+        <button type="button" onClick={(e) => this.eventHandler(e.target.textContent)} className="orange">-</button>
+        <button type="button" onClick={(e) => this.eventHandler(e.target.textContent)}>1</button>
+        <button type="button" onClick={(e) => this.eventHandler(e.target.textContent)}>2</button>
+        <button type="button" onClick={(e) => this.eventHandler(e.target.textContent)}>3</button>
+        <button type="button" onClick={(e) => this.eventHandler(e.target.textContent)} className="orange">+</button>
+        <button type="button" onClick={(e) => this.eventHandler(e.target.textContent)} className="zero">0</button>
+        <button type="button" onClick={(e) => this.eventHandler(e.target.textContent)}>.</button>
+        <button type="button" className="orange" onClick={(e) => this.eventHandler(e.target.textContent)}>=</button>
```

Comment:
- [ ] (Optional) It would be nice to the mathematical symbol/numbers in an array then render each one of them dynamically using the array `map` function. You can read this [article](https://www.pluralsight.com/guides/how-to-use-the-map()-function-to-export-javascript-in-react) to get a better understanding of what I am saying.